packages/host/android/app/build.gradle

```
@@ -93,13 +93,13 @@ android {
 
     compileSdkVersion rootProject.ext.compileSdkVersion
 
-    namespace "com.host"
+    namespace "com.callstack.superappshowcase.host"
     defaultConfig {
-        applicationId "com.host"
+        applicationId "com.callstack.superappshowcase.host"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 1
-        versionName "1.0"
+        versionCode 2
+        versionName "0.0.1"
```

Maybe let's start with v1 so reviewers don't have doubts about this being a "prerelease" or something
```suggestion
        versionName "1.0.0"
```

---

Android is already released with that version and passed the review without any issues. I think we are kind of stuck with `0.0.1` unless we want to rerelease Android just to bump the version and use that for the iOS review process